mathics/builtin/atomic/numbers.py

```diff
@@ -145,6 +145,74 @@ def convert_float(x, base, exponents):
         raise TypeError(x)
 
 
+class Accuracy(Builtin):
+    """
+    <dl>
+      <dt>'Accuracy[$x$]'
+      <dd>examines the number of significant digits of $expr$ to the right of the the decimal point in the number x.
+    </dl>
+    This is rather a proof-of-concept than a full implementation.
+
+
+    Accuracy of a real number is estimated from its value and its precision:
+
+    >> Accuracy[3.1416`2]
+     = 1.50298
+
+    Notice that the value is not exactly equal to the obtained in WMA: This is due to the different way in which
+    Precision is handled in Sympy.
```

Comment:
Sorry for another small thing: we should start us capitalize SymPy the way it was recommended, with a capital "P"